lib/fluent_plugin_kinesis/version.rb

```diff
@@ -13,5 +13,5 @@
 # language governing permissions and limitations under the License.
 
 module FluentPluginKinesis
-  VERSION = '3.4.1'
+  VERSION = '3.4.2'
```

Comment:
We should handle adding feature and version upgrade separately. Can you remove this version upgrade to 3.4.2 from your PR?

Reply:
Sure, changes reverted